### ISSUE TITLE
MacOS 12.3

### ISSUE THREAD

**Issue by yannpub** [osxfuse/osxfuse#873]
I've read here and there that kernel extensions for remote files will mostly be deprecated into the next version of MacOS 12.3
Dropbox, Onedrive and most probably other will be impacted. Will MacFuse continue to work with MacOS12.3, as it also use kernel extensions ?

**Comment by bfleischer** [comment]
Apple is deprecating / phasing out the private com.apple.fileutils API that is (or was) used by Microsoft OneDrive and Dropbox. To my knowledge this is not about kernel extensions. OneDrive and Dropbox do not ship kernel extensions. The com.apple.fileutils API seems to predate the File Provider API and was made available exclusively to Microsoft and Dropbox. This means, only OneDrive and Dropbox will be affected by the changes in macOS 12.3.

**TL;DR**

I can confirm that macFUSE works just fine on the first macOS 12.3 beta that Apple released yesterday.
